tests/test/import/test.sh

```diff
@@ -160,11 +160,7 @@ rlJournalStart
         rlRun -s 'tmt test import --makefile --no-nitrate --no-purpose' '0'
         rlAssertExists "main.fmf"
         rlRun -s 'cat main.fmf'
-        rlAssertGrep 'test: |-' "main.fmf"
-        rlAssertGrep '( export PS4=.* \\' "main.fmf"
-        rlAssertGrep 'chmod +x runtest.sh; \\' "main.fmf"
-        rlAssertGrep './runtest.sh; \\' "main.fmf"
-        rlAssertGrep 'chmod -x runtest.sh )' "main.fmf"
+        rlRun 'yq .test main.fmf | grep "( export PS4=.*chmod -x runtest.sh )"'
```

Comment:
Please assert whole content of `test` attribute.
We need to make sure tmt stores everything (this test would pass even if tmt skips 2 lines of the original makefile).

Reply:
@lukaszachy, thank you for your quick review. The related checking is updated, please help to take a look again, thanks!